lib/cn/cn.c

```diff
@@ -65,6 +70,8 @@
 #include "cn_perfc.h"
 #include "kvset_internal.h"
 
+#define ENDPOINT_FMT_CN_TREE "/kvdbs/%s/kvs/%s/cn/tree"
```

Comment:
You don't want these centralized?

Reply:
No, I want them to live where they are used. In this case, this is the file where the route is now registered.